model/project_event.go

```diff
@@ -81,7 +81,7 @@ func (e *ProjectChangeEventEntry) SetBSON(raw mgobson.Raw) error {
 	case primitive.ObjectID:
 		e.ID = v.Hex()
 	default:
-		return errors.Errorf("unrecognized ID format for event %v", v)
+		return errors.Errorf("unrecognized ID format for event %T", v)
```

Comment:
I think probably not since it's a stringified Go type, which can't have spaces. My basic rule is that strings have quotes (except maybe in special cases like string durations).